File: c course/array class questions/2d/q5.c
#include<stdio.h>
int main()
{
int size,i,j;
printf("enter the size of array:");
scanf("%d",&size);
int x[size][size];
printf("enter the element of array:");
for(i=0;i<size;i++){
    for(j=0;j<size;j++){
        scanf("%d",&x[i][j]);
    }
}

for(i=0;i<size;i++)
{
    for(j=0;j<size;j++)
    {
        printf("%d ",x[j][i]);
    }
    printf("\n");
}

return 0;
}

File: java/W3 resourse java/Array/q5.java
import java.util.Scanner;

public class q5 {
    public static void main(String[] args){
        int []arr = {5,9,2,0,1,7};
        Scanner in = new Scanner(System.in);
        System.out.print("Enter the number you want to check : ");
        int num = in.nextInt();
        Boolean boo = false;
        for(int i = 0; i<arr.length;i++ ){
            if (num == arr[i]){
                boo = true;
                break;
            }        
        }
       if (boo == true){
        System.out.println("Element is present " );
        
       }
       else {
        System.out.println("Element is not present");
       }
        
    }
}
